openpype/hosts/max/api/lib.py

```diff
@@ -314,6 +345,54 @@ def set_timeline(frameStart, frameEnd):
     return rt.animationRange
 
 
+def reset_colorspace():
+    """OCIO Configuration
+    Supports in 3dsMax 2024+
+
+    """
+    if int(get_max_version()) < 2024:
+        return
+    project_name = get_current_project_name()
+    colorspace_mgr = rt.ColorPipelineMgr
+    colorspace_mgr.Mode = rt.Name("OCIO_Custom")
+    ocio_config_path = os.environ.get("OCIO")
+    global_imageio = get_project_settings(
+        project_name)["global"]["imageio"]
+    if global_imageio["activate_global_color_management"]:
+        ocio_config = global_imageio["ocio_config"]
+        ocio_config_path = ocio_config["filepath"][-1]
+
+    max_imageio = get_project_settings(
+        project_name)["max"]["imageio"]
+    if max_imageio["activate_host_color_management"]:
+        ocio_config = max_imageio["ocio_config"]
+        if ocio_config["override_global_config"]:
+            ocio_config_path = ocio_config["filepath"][0]
+            if not ocio_config_path:
+                # use the default ocio config path instead
+                ocio_config_path = ocio_config["filepath"][-1]
```

Comment:
You are retrieving the full project settings twice - re-use it please.

Also, shouldn't `colorspace.get_imageio_config` be doing this to get the config you need to apply instead? @jakubjezek001 


```python
from openpype.pipeline import colorspace

def reset_colorspace():
    """OCIO Configuration
    Supports in 3dsMax 2024+
    """
    if int(get_max_version()) < 2024:
        return

    config_data = colorspace.get_imageio_config(project_name, 
                                                "max", 
                                                project_settings)
    if config_data:
        path = config_data["path"]    
        colorspace_mgr = rt.ColorPipelineMgr
        colorspace_mgr.Mode = rt.Name("OCIO_Custom")
        colorspace_mgr.OCIOConfigPath = ocio_config_path
```

Reply:
I have tested it from my side, this code works.